.github/CODEOWNERS

```diff
@@ -1,2 +1,3 @@
 # Default owners for all files in this repository
-* @itdependsnetworks
\ No newline at end of file
+* @itdependsnetworks
```

Comment:
I think this is meant to be on a single line
https://github.com/networktocode/diffsync/blob/master/.github/CODEOWNERS#L2

Reply:
Okay, I've done it both ways and if you dont specify file path it assumes default ownership.  I'll get it corrected though to make sure we're in aligned throughout our repos.